### ISSUE TITLE
ci: ERROR collecting tests/unit/discovery/py_info/test_py_info_exe_based_of.py 

### ISSUE THREAD

**Issue by ssbarnea** [pypa/virtualenv#1633]
I seen error below on CI and seems totally unrelated to the change being tested.

https://dev.azure.com/pypa/virtualenv/_build/results?buildId=19019&view=logs&jobId=cc0564e1-d822-5f24-0bbb-f60430a68e65&j=cc0564e1-d822-5f24-0bbb-f60430a68e65&t=9fc317a6-b66c-593b-646c-b1040cd24f7e

```
==================================== ERRORS ====================================
__ ERROR collecting tests/unit/discovery/py_info/test_py_info_exe_based_of.py __
tests/unit/discovery/py_info/test_py_info_exe_based_of.py:20: in <module>
    BASE = {str(Path(CURRENT.executable).parent.relative_to(Path(CURRENT.prefix))), "."}
/usr/local/lib/python2.7/site-packages/pathlib2/__init__.py:1132: in relative_to
    .format(str(self), str(formatted)))
E   ValueError: '/usr/local/opt/python@2/bin' does not start with '/usr/local/Cellar/python@2/2.7.17_1/Frameworks/Python.framework/Versions/2.7'
- generated xml file: /Users/runner/runners/2.164.8/work/1/s/.tox/junit.py27-lower.xml -
==================================== ERRORS ====================================
__ ERROR collecting tests/unit/discovery/py_info/test_py_info_exe_based_of.py __
tests/unit/discovery/py_info/test_py_info_exe_based_of.py:20: in <module>
    BASE = {str(Path(CURRENT.executable).parent.relative_to(Path(CURRENT.prefix))), "."}
/usr/local/lib/python2.7/site-packages/pathlib2/__init__.py:1132: in relative_to
    .format(str(self), str(formatted)))
E   ValueError: '/usr/local/opt/python@2/bin' does not start with '/usr/local/Cellar/python@2/2.7.17_1/Frameworks/Python.framework/Versions/2.7'
- generated xml file: /Users/runner/runners/2.164.8/work/1/s/.tox/junit.py27-lower.xml -
==================================== ERRORS ====================================
__ ERROR collecting tests/unit/discovery/py_info/test_py_info_exe_based_of.py __
tests/unit/discovery/py_info/test_py_info_exe_based_of.py:20: in <module>
    BASE = {str(Path(CURRENT.executable).parent.relative_to(Path(CURRENT.prefix))), "."}
/usr/local/lib/python2.7/site-packages/pathlib2/__init__.py:1132: in relative_to
    .format(str(self), str(formatted)))
E   ValueError: '/usr/local/opt/python@2/bin' does not start with '/usr/local/Cellar/python@2/2.7.17_1/Frameworks/Python.framework/Versions/2.7'
- generated xml file: /Users/runner/runners/2.164.8/work/1/s/.tox/junit.py27-lower.xml -
==================================== ERRORS ====================================
__ ERROR collecting tests/unit/discovery/py_info/test_py_info_exe_based_of.py __
tests/unit/discovery/py_info/test_py_info_exe_based_of.py:20: in <module>
    BASE = {str(Path(CURRENT.executable).parent.relative_to(Path(CURRENT.prefix))), "."}
/usr/local/lib/python2.7/site-packages/pathlib2/__init__.py:1132: in relative_to
    .format(str(self), str(formatted)))
E   ValueError: '/usr/local/opt/python@2/bin' does not start with '/usr/local/Cellar/python@2/2.7.17_1/Frameworks/Python.framework/Versions/2.7'
- generated xml file: /Users/runner/runners/2.164.8/work/1/s/.tox/junit.py27-lower.xml -
```

For sure I was not ablet to reprodue the same error locally, on same platform.

**Comment by gaborbernat** [comment]
I'm fairly certain this is our python 2 macos bugs manifestation 👍

**Comment by gaborbernat** [comment]
Confirmed; it's duplicate of https://github.com/pypa/virtualenv/issues/1561

**Comment by gaborbernat** [comment]
Hello, a fix for this issue has been released via virtualenv 20.0.5; see https://pypi.org/project/virtualenv/20.0.5/ (https://virtualenv.pypa.io/en/latest/changelog.html#v20-0-5-2020-02-21) . Please give a try and report back if your issue has not been addressed; if not, please comment here, and we'll reopen the ticket. We want to apologize for the inconvenience this has caused you and say thanks for having patience while we resolve the unexpected bugs with this new major release.

![thanks](https://media.tenor.co/images/1114067881329bb4218880eeb7ec34fd/tenor.gif)
